module-services/service-bluetooth/ServiceBluetooth.cpp

```diff
@@ -45,7 +45,7 @@ namespace
 {
     constexpr auto BluetoothServiceStackDepth = 2560U;
     inline constexpr auto nameSettings        = "ApplicationSettings";
-    inline constexpr auto connectionTimeout   = std::chrono::minutes{30};
+    inline constexpr auto connectionTimeout   = std::chrono::minutes{10};
```

Comment:
was this discussed? :P

Reply:
Yes, it was discussed and accepted by @arogucka as stated in the comment PR description :)